src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace SMG\ManagerBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\Controller\Annotations;
+use SMG\UserBundle\Entity\User;
+use FOS\RestBundle\View\View;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class ManagersController extends FOSRestController
+{
+    use Traits\TokenFromHeaderTrait;
+    use Traits\HandleUserTrait;
+
+    /**
+     * TODO: annotation actually required to make difference
+     * with the other API which has the same name
+     *
+     * @Annotations\Post("/admin/users")
+     *
+     * @ParamConverter("user", converter="fos_rest.request_body")
+     */
+    public function postUserAction(User $user)
+    {
+        $this->throwIfClientNot('backend');
+        $manager = $this->get('fos_user.user_manager');
```

Comment:
manager is not semantically linked with the line above, so it make no sense that they're part of the same block of contigous line,  on the other hand it is used by the line below the todo, so they should be together